tests/UnifiedSpecTests/index-management/createSearchIndexes.json

```diff
@@ -0,0 +1,169 @@
+{
+  "description": "createSearchIndexes",
+  "schemaVersion": "1.4",
+  "createEntities": [
+    {
+      "client": {
+        "id": "client0",
+        "useMultipleMongoses": false,
+        "observeEvents": [
+          "commandStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "database0",
+        "client": "client0",
+        "databaseName": "database0"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection0",
+        "database": "database0",
+        "collectionName": "collection0"
+      }
+    }
+  ],
+  "runOnRequirements": [
+    {
+      "minServerVersion": "7.0.0",
+      "topologies": [
+        "replicaset",
+        "load-balanced",
+        "sharded"
+      ],
+      "serverless": "forbid"
+    }
+  ],
+  "tests": [
+    {
+      "description": "empty index definition array",
```

Comment:
Some tests fail because the the exception is thrown by PHP before sending the operation to the server.

Reply:
It may be worth asking the DBX spec test authors whether they considered using `isClientError` here. But assuming that's unspecified and drivers are free to validate this before sending a command or rely on the server to report an error, I'm still not sure why this would fail.

IIRC, [`isError: true`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#expectederror) in the test format shouldn't distinguish about the type of error. Server-side and client-side errors should both satisfy it. If that's not the case, perhaps there's a bug with out test runner implementation.

Reply:
> I'm still not sure why this would fail.

Because it's not Atlas, the command is not supported by non-Atlas server. The actual assertion is in `expectEvents`.

Reply:
Ah, is this something that will ultimately be covered by adding the end-to-end testing? I suppose we'll need some new syntax to skip these tests on non-Atlas servers, then.

Reply:
If we can run the tests on Atlas from Github Actions, this tests on failing commands could be removed from the spec repository.